src/MSBuild/XMake.cs

```diff
@@ -3423,27 +3438,33 @@ string unquotedParameter
             }
             catch (IOException e)
             {
-                InitializationException.Throw("LoggerCreationError", unquotedParameter, e, false);
+                HandleLoggerCreationError(verbosity, unquotedParameter, optional, e, messageResource: "LoggerCreationError", showStackTrace:false);
```

Comment:
Nit: could collapse all the multiple catch clauses into one with conditional exceptions